### PR TITLE
More tests

### PR DESCRIPTION
**Pull request by Suedie**
A few lines are not covered

### CODE REVIEW

**Review by pkasinski — APPROVED** [review]
Looks good!

**Review by rojtekin — APPROVED** [review]
LGTM;

File: core/src/model/IPlayerCharacter.java [hunk 1]
```diff
@@ -52,6 +55,8 @@ public interface IPlayerCharacter {
      */
     void addTweak(Set<Tweak> tweaks);
 
+    Map<LivingTrait, ArrayList<Tweak>> getTweaks();
```
Comment (rojtekin) [inline core/src/model/IPlayerCharacter.java:36]: why is this here?

File: core/src/model/LivingEntity.java [hunk 2]
```diff
@@ -126,7 +129,7 @@ public void move(Direction direction, float speed) {
     }
 
     /**
-     * Damages every touched hostile entity
+     * Attacks every touched entity
```
Comment (rojtekin) [inline core/src/model/LivingEntity.java]: why is the comment changed? it doesnt do dmg to everyone only hostile ones?

File: core/src/model/LivingEntity.java [hunk 3]
```diff
@@ -135,6 +138,13 @@ private void damageTouched(Collisions projectedCollisions) {
         }
     }
 
+    /**
+     * When a living entity is attacked by someone of a different faction
+     * it will take damage. If it is attacked by a "friendl" entity of the same
```
Comment (rojtekin) [inline core/src/model/LivingEntity.java]: typo on friendl

File: core/src/model/PlayerCharacter.java [hunk 4]
```diff
@@ -54,6 +54,12 @@ public void addTweak(Set<Tweak> tweaks) {
         }
     }
 
+    @Override
+    public Map<LivingTrait, ArrayList<Tweak>> getTweaks() {
```
Comment (rojtekin) [inline core/src/model/PlayerCharacter.java:52]: where is this necessary?

**Review by rojtekin — COMMENTED** [review]
im confused about some code feels like its unecessary

File: core/src/model/PlayerCharacter.java (hunk 4 above)
Comment (Suedie) [reply core/src/model/PlayerCharacter.java:52]: It's only necessary for testing

File: core/src/model/LivingEntity.java (hunk 2 above)
Comment (Suedie) [reply core/src/model/LivingEntity.java]: Technically it only calls the beAttacked method on every touched entity. The entities determine themselves what the effect of being attacked should be.

File: core/src/model/IPlayerCharacter.java (hunk 1 above)
Comment (Suedie) [reply core/src/model/IPlayerCharacter.java:36]: Only for testing purposes
